pages/apis/webhooks/agent_events.md.erb

```diff
@@ -8,23 +8,23 @@
 <tbody>
   <tr>
     <th><code>agent.connected</code></th>
-    <td>An agent has connected</td>
+    <td>An agent has connected to the API</td>
   </tr>
   <tr>
     <th><code>agent.lost</code></th>
-    <td>An agent has been marked as lost  </td>
+    <td>An agent has been marked as lost. This happens when Buildkite stops receiving heartbeats from the agent.</td>
   </tr>
   <tr>
     <th><code>agent.disconnected</code></th>
-    <td>An agent has disconnected</td>
+    <td>An agent has disconnected. This happens when the agent disconnects from the API.</td>
```

Comment:
and is different from having no internet connection, say?

Reply:
Or perhaps below is clearer, to distinguish between stopping (API tells it what to do) and disconnecting (agent decides what to do)

```suggestion
    <td>An agent has disconnected. This happens when the agent shuts down and disconnects from the API.</td>
```